blocks.config.json

```diff
@@ -127,6 +127,16 @@
     "matches": ["*.md", "*.mdx"],
     "example_path": "https://github.com/githubnext/blocks-tutorial/blob/main/README.md"
   },
+  {
+    "type": "file",
+    "id": "use-case-block",
+    "title": "Blocks use case",
+    "description": "View Blocks use case writeups",
+    "sandbox": false,
```

Comment:
does this work in `blocks`? I thought MDX did some stuff that violates the CSP in `blocks`, which is why we have the `live-markdown` block sandboxed.

Happy to be wrong, in which case we could un-sandbox `live-markdown` too.

Reply:
yeah great question! I was wondering about that but it works locally and I wanted to see if it worked in prod. but now that i think of it, it might need the `script-src 'unsafe-eval'` CSP, which we're setting in dev